sdk/eventhub/Azure.Messaging.EventHubs/Directory.Build.props

```diff
@@ -1,6 +1,20 @@
-﻿<Project ToolsVersion="Current" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+﻿<Project ToolsVersion="Current" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">  
+  <!-- 
+    Identify samples as a test support project to keep the target framework synchronized, since both are 
+    based on a version of netcoreapp. 
+    
+    Because the common SDK build properties need this value in order to recognize the desired
+    target frameworks and related, this needs to be set before the common properties import.
+  -->
+  <PropertyGroup Condition="$(MSBuildProjectName.EndsWith('.Samples'))">
```

Comment:
I had to do something "similar" for appconfig and keyvault. if we are doing it for various projects,
should we just add this to https://github.com/Azure/azure-sdk-for-net/blob/master/eng/Directory.Build.Data.props#L21 ?
@weshaggard

Reply:
Yes I think it is worth moving it to the common location for Samples.